server/BUILD

```diff
@@ -81,6 +81,9 @@ rust_library(
         ":development_mode": [],
         ":published_mode": ["published"],
     }),
+    tags = [
+        "declared-features=published",
+    ]
```

Comment:
Not super happy with `declared-features` as a tag name, but I can't come up with anything better right now. I can see how `features=published` here would be confusing.